core/src/test/scala/stryker4s/stubs/TestProcessRunner.scala

```diff
@@ -5,11 +5,11 @@ import stryker4s.run.process.ProcessRunner
 
 import scala.util.Try
 
-class TestProcessRunner(returns: Try[Int]) extends ProcessRunner {
+class TestProcessRunner(returns: Try[Int]*) extends ProcessRunner {
   val timesCalled: Iterator[Int] = Iterator.from(0)
 
   override def apply(command: String, workingDir: File, envVar: (String, String)): Try[Int] = {
     timesCalled.next()
-    returns
+    returns(envVar._2.toInt - 1)
```

Comment:
What does this do? Does it return the envVar as the result for the exit code? Maybe a comment would be nice

Reply:
Added some comments on how this works.

What do you think about starting the test run numbers on 0 instead of 1? This would allow us to remove the -1 in the test run.

Reply:
Hmm, maybe. In my mind mutation run 0 was always "reserved" for no mutations. But I don't see why not. What do StrykerJS and Stryker.NET do? I think it's good to have it the same as them

Reply:
Ah, I see. Couldn't we just use `ACTIVE_MUTANT=''` to clear the env variable? Indeed, just start at 0 like decent human beings.